cmd/kpod/unpause.go

```diff
@@ -24,33 +24,30 @@ var (
 )
 
 func unpauseCmd(c *cli.Context) error {
+	runtime, err := getRuntime(c)
+	if err != nil {
+		return errors.Wrapf(err, "could not get runtime")
+	}
+	defer runtime.Shutdown(false)
+
 	args := c.Args()
 	if len(args) < 1 {
 		return errors.Errorf("you must provide at least one container name or id")
 	}
 
-	config, err := getConfig(c)
-	if err != nil {
-		return errors.Wrapf(err, "could not get config")
-	}
-	server, err := libkpod.New(config)
-	if err != nil {
-		return errors.Wrapf(err, "could not get container server")
-	}
-	defer server.Shutdown()
-	if err := server.Update(); err != nil {
-		return errors.Wrapf(err, "could not update list of containers")
-	}
 	var lastError error
-	for _, container := range c.Args() {
-		cid, err := server.ContainerUnpause(container)
+	for _, arg := range args {
+		ctr, err := runtime.LookupContainer(arg)
 		if err != nil {
+			return errors.Wrapf(err, "error looking up container %q", arg)
```

Comment:
This also needs the lastError dance (lines 45-48)

Reply:
fixed